r/NEWS.md

```diff
@@ -19,12 +19,65 @@
 
 # arrow 0.16.0.9000
 
+## Feather v2
+
+This release includes support for version 2 of the Feather file format.
+Feather v2 features full support for all Arrow data types, and it allows
+files to be compressed using either `lz4` or `zstd`.
+`write_feather()` can write either version 2 or
+[version 1](https://github.com/wesm/feather) Feather files, and `read_feather()`
+automatically detects which file version it is reading.
+
+Related to this change, several functions around reading and writing data
+have been reworked. `read_ipc_stream()` and `write_ipc_stream()` have been
+added to facilitate writing data to the Arrow IPC stream format, which is
+slightly different from the IPC file format (Feather v2 *is* the IPC file format).
+
+Behavior has been standardized: all `read_<format>()` return an R `data.frame`
+(default) or a `Table` if the argument `as_data_frame = FALSE`;
+all `write_<format>()` functions return the data object, invisibly.
+To facilitate some workflows, a special `write_to_raw()` function is added
+to wrap `write_ipc_stream()` and return the `raw` vector containing the buffer
+that was written.
+
+To achieve this standardization, `read_table()`, `read_record_batch()`,
+`read_arrow()`, and `write_arrow()` have been deprecated.
+
+## Python interoperability
+
+The 0.17 Apache Arrow release includes a C data interface that allows you to
+share or pass data from one runtime to another without copying it. This enables
```

Comment:
This might provoke comments like "Wait, couldn't Arrow already do that?" 

I would say that the "C data face allows exchanging Arrow data in-memory runtime at the C level without copying and without either project having an build or runtime dependency on the other". The real game changer is the lack of any dependency / coupling, code or otherwise. 

The key differentiators with the C interface to help people not get confused:

* Exchanging data *in-process* from one library/function to another
* Projects have zero knowledge of each other except for their use of the C interface
* Neither project is required to depend on the other at build-time or run-time

By contrast, the IPC format:

* Truly language-independent (e.g. does not presume any C ABI)
* Allows exchanging data *between* processes
* Can be stored on disk or transferred over TCP